yaksh/views.py

```diff
@@ -1543,7 +1543,10 @@ def new_activation(request, email=None):
         user.profile.key_expiry_time = timezone.now() + \
                                 timezone.timedelta(minutes=20)
         user.profile.save()
-        success, msg = send_user_mail(user.email, user.profile.activation_key)
+        new_user_data = User.objects.get(email=email)
+        success, msg = send_user_mail(new_user_data.email,
+                                      new_user_data.profile.activation_key
```

Comment:
Again, <code>new_user_data.profile.activation_key</code> is the same as <code>user.profile.activation_key</code>